community-operators/microcks/microcks-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,197 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: microcks-operator.v0.1.0
+  namespace: REPLACE_NAMESPACE
+  annotations:
```

Comment:
Additional annotations for UI rendering can be found here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md

Reply:
CSV has been completed / normalized in commit e49983a